mobify/astro-scaffold
# [HYB-691] Cart Modal

Cart Modal for both iOS and Android

JIRA: [**[HYB-691] As a scaffold user, I want a cart in a modal**](https://mobify.atlassian.net/browse/HYB-691)
Linked PRs: **None**

## Changes
- Placed Cart in modal for both iOS and Android
- Wired back button for android to dismiss cart when it is visible
- Centralized configuration for cart & added a baseConfig file to host app-wide configurables

## How to test-drive this PR
- Launch app on Android and iOS make sure they show up and dismiss properly
- On Android make sure you can dismiss with hardware back button

## TODOS:
- ~~[ ] Tests are passing on your machine.~~
